cns/restserver/internalapi.go

```diff
@@ -157,13 +157,14 @@ func (service *HTTPRestService) SyncHostNCVersion(ctx context.Context, channelMo
 	if err != nil {
 		logger.Errorf("sync host error %v", err)
 	}
-	syncHostNCVersion.WithLabelValues(strconv.FormatBool(err == nil)).Observe(time.Since(start).Seconds())
+	syncHostNCVersionCount.WithLabelValues(strconv.FormatBool(err == nil)).Inc()
```

Comment:
Not sure if we inc non err label, how do we set alert? How do we know that there is an error with syncHostNCVersion?

Reply:
we have this label "ok", and it will be set to `0` or `1` based on `err == nil`, and we will increment the correct series based on that. 
the alert would be based on the value of `sync_host_nc_version_total{"ok=0"}[15m]` or something like that.